### ISSUE TITLE
how to use sails.session ?

### ISSUE THREAD

**Issue by zieglar** [balderdashy/sails#2854]
I want to use global session to record something, I try to log sails.session and sails.config.session, totally different object. so ,what can I use? and where is session usage document?
sails.session==>
```
{ defaults: { session: { adapter: 'memory', key: 'sails.sid' } },
  configure: [Function],
  initialize: [Function],
  generateNewSidCookie: [Function],
  parseSessionIdFromCookie: [Function],
  get: [Function],
  set: [Function],
  identity: 'session',
  configKey: 'session',
  config: { envs: [] },
  middleware: {},
  routes: { before: {}, after: {} } }
```
sails.config.session==>
```
{ adapter: 'connect-redis',
  host: 'localhost',
  port: '6379',
  db: '8',
  prefix: 'lds:',
  ttl: 691200
  ...and more
}
```

**Comment by tjwebb** [comment]
> I want to use global session to record something

Can you offer an example of what you're trying to do?

**Comment by zieglar** [comment]
@tjwebb for example
One service need use id and secret to get a token and token will expirein five day.
And service do not allow application have frequent request token.
So I need cache this token and expirein, when expire I can know this token want request again.
like laravel's cache http://laravel.com/docs/5.0/cache

**Comment by LeonardoGentile** [comment]
`req.session` is not documented in the website either under Concept or Reference. Would be nice to have a section dedicated to explain how the session works/how to use it, as well as the reference about `req.session`

**Comment by sailsbot** [comment]
Thanks for posting, @zieglar.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
